modules.d/35network-manager/nm-lib.sh

```diff
@@ -14,6 +14,7 @@ nm_generate_connections() {
             /etc/sysconfig/network-scripts/ifcfg-*; do
             [ -f "$i" ] || continue
             echo '[ -f /tmp/nm.done ]' > "$hookdir"/initqueue/finished/nm.sh
+            > /run/NetworkManager/initrd/neednet # activate nm-run.service
```

Comment:
You need to change this line to 

`: > /run/NetworkManager/initrd/neednet`

Reply:
done :)